README.md

```diff
@@ -270,9 +270,13 @@ These are the available config options for making requests. Only the `url` is re
   // `xsrfHeaderName` is the name of the http header that carries the xsrf token value
   xsrfHeaderName: 'X-XSRF-TOKEN', // default
 
-  // `progress` allows handling of progress events for 'POST' and 'PUT uploads'
-  // as well as 'GET' downloads
-  progress: function (progressEvent) {
+  // `progressUpload` allows handling of progress events for uploads
+  progressUpload: function (progressEvent) {
```

Comment:
I would suggest to rename `progressUpload` and 'progressDownload` to `onUploadProgress` and `onDownloadProgress` respectively.